RedBugs_webapp/src/main/java/com/isa/pl/redbugs/service/PathFindingService.java

```diff
@@ -1,31 +1,48 @@
 package com.isa.pl.redbugs.service;
 
+import com.isa.pl.redbugs.model.Route;
 import com.isa.pl.redbugs.model.Stop;
 import com.isa.pl.redbugs.service.pathfinding.DistanceScorer;
 import com.isa.pl.redbugs.service.pathfinding.Graph;
 import com.isa.pl.redbugs.service.pathfinding.RouteFinder;
 import org.springframework.stereotype.Service;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+
+import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 @Service
 public class PathFindingService {
 
     public Graph<Stop> setUpStopsAndConnections() throws Exception {
-        Set<Stop> stops = InitDataService.stopsDataList().stream().collect(Collectors.toSet());
+        Set<Stop> stops = new HashSet<>(InitDataService.stopsDataList());
+        Set<Route> routes = new HashSet<>(InitDataService.routesDataList());
```

Comment:
here we can get data from .json file

ReadService readService = new ReadService();
        Set<Stop> stops = new HashSet<>(readService.readJson("Stops.json", Stop[].class));
        Set<Route> routes = new HashSet<>(readService.readJson("Routes.json", Route[].class));